imagebuildah/stage_executor.go

```diff
@@ -253,7 +253,7 @@ func (s *StageExecutor) volumeCacheRestore() error {
 // don't care about the details of where in the filesystem the content actually
 // goes, because we're not actually going to add it here, so this is less
 // involved than Copy().
-func (s *StageExecutor) digestSpecifiedContent(node *parser.Node, argValues []string) (string, error) {
+func (s *StageExecutor) digestSpecifiedContent(node *parser.Node, argValues []string, envValues []string) (string, error) {
```

Comment:
Would it be better to just pass into argvalues 
append(argValued, envValues ...)
Rather then add a new param?

Reply:
I was on the fence and decided it would be better to have one append within the function.  That way future callers (most likely me) wouldn't forget to append both as the param would be in their face, and the append is only in once place this way.